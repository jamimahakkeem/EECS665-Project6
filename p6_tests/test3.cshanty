void main() {
	int a;
	a = 5;
	a = a + 1;
	a = a - 1;
	a = a * 1;
	a = a / 1;
}