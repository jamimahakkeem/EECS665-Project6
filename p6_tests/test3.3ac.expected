[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
a (local var of 8 bytes)
tmp0 (tmp var of 8 bytes)
tmp1 (tmp var of 8 bytes)
tmp2 (tmp var of 8 bytes)
tmp3 (tmp var of 8 bytes)
[END main LOCALS]
main:       enter main
            [a] := 5
            [tmp0] := [a] ADD64 1
            [a] := [tmp0]
            [tmp1] := [a] SUB64 1
            [a] := [tmp1]
            [tmp2] := [a] MULT64 1
            [a] := [tmp2]
            [tmp3] := [a] DIV64 1
            [a] := [tmp3]
lbl_0:      leave main
